_includes/pbc/all/install-features/202304.0/install-the-product-labels-promotions-and-discounts-feature.md

```diff
@@ -0,0 +1,68 @@
+
```

Comment:
* As I remember we have to add a sentence like this to the header of the IG:
This document describes how to integrate the  Product Labels + Promotions & Discounts Feature into a Spryker project.